src/GitPrompt.ps1

```diff
@@ -69,6 +69,14 @@ function Write-Prompt {
         $StringBuilder
     )
 
+    if (!$Object -or ($Object -is [PoshGitTextSpan] -and !$Object.Text)) {
+        if ($StringBuilder) {
+            return $StringBuilder
+        }
+
+        return ""
```

Comment:
Could probably simplify this to a one-liner: `return $(if ($StringBuilder) {$StringBuilder} else {""})`

Reply:
That would be fine. I'm crashing for the night, but if you want to patch these up I can merge in the morning.

Reply:
Will do.